### DISCUSSION TITLE
GitHub request error  'Cookies must be enabled to use GitHub'

### DISCUSSION THREAD

**Discussion by syg-ygormarcelino** [community/community#55888] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/52581546/2f8af919-257b-4621-88da-2caf81126a4f)

I'm facing the error 'Cookies must be enabled to use GitHub.' when trying to retrieve the access_token with the request 'https://github.com/oauth/access_token'

Does anyone know what can it be?

**Comment by JoaquinRodriguezc** [comment]
I got the same error
